docs/src/tut/rps/index.md

```diff
@@ -1835,7 +1835,8 @@ stdlib.setWalletFallback(stdlib.walletFallback({
 
 Or, you could have your users use [WalletConnect](https://walletconnect.com/) to connect to the [Algorand Wallet](https://algorandwallet.com/), by adding the following:
 ```js
-import WalletConnect from '@reach-sh/stdlib/ALGO_WalletConnect';
+import * as reachsdk from '@reach-sh/stdlib';
+const WalletConnect = reachsdk.ALGO_WalletConnect;
```

Comment:
See above

Reply:
[b0b8b5d](https://github.com/reach-sh/reach-lang/pull/664/commits/b0b8b5d7c8b6ac33868b17c5f5503189e1ea9701) addresses this.